packages/store/addon/-private/system/ds-model-store.ts

```diff
@@ -245,9 +245,13 @@ class Store extends CoreStore {
     }
   }
 
-  _attributesDefinitionFor(modelName: string) {
+  _attributesDefinitionFor(modelName: string, identifier?: StableRecordIdentifier) {
     if (CUSTOM_MODEL_CLASS) {
-      return this.getSchemaDefinitionService().attributesDefinitionFor(modelName);
+      if (identifier) {
```

Comment:
I think this should also be guarded by `IDENTIFIERS`, right? Or is that not needed because `CUSTOM_MODEL_CLASS` itself fundamentally requires `IDENTIFIERS`?

Reply:
Not needed cause CUSTOM_MODEL_CLASS requires IDENTIFIERS